src/ui/utils/tokenManager.tsx

```diff
@@ -212,7 +224,7 @@ class TokenManager {
     try {
       return await this.auth0Client.getAccessTokenSilently({ audience, ignoreCache: refresh });
     } catch (e: any) {
-      if (e.error !== "login_required" && e.error !== "consent_required") {
```

Comment:
`login_required` is the expected response if the user was not authenticated when we call this under the new circumstances of this PR so we'll let that be thrown and caught above.